mkt/comm/templates/comm/emails/escalation_vip.html

```diff
@@ -3,9 +3,9 @@
 {% block content %}
 Dear Superheroes,
 
-A developer has uploaded/resubmitted an app we've designated to be super important and/or a massive PITA.
+A developer has uploaded/resubmitted an {{ obj_type }} we've designated to be super important and/or a massive PITA.
```

Comment:
The article here doesn't work in both cases: 

- `an app`
- `an Firefox OS add-on`